lib/Chat/SystemMessage/Listener.php

```diff
@@ -253,6 +253,10 @@ protected function sendSystemLobbyMessage(LobbyModifiedEvent $event): void {
 			return;
 		}
 
+		if ($event->getRoom()->isFederatedConversation()) {
```

Comment:
Should we move this to the `handle()` function so we always skip it, instead of adding it on each method over and over again? Basically the whole listener should never create a message if its a federated convo

Reply:
Sounds good :+1:

Reply:
Added in #12777